python/requirements/ml/tune-requirements.txt

```diff
@@ -7,9 +7,7 @@ bayesian-optimization==1.4.3
 ConfigSpace==0.7.1
 hpbandster==0.7.4
 
-# Todo: Use version pin again after HEBO is updated on pypi
-git+https://github.com/huawei-noah/HEBO@9a2a674c22518eed35a8b98e5134576741a95410#subdirectory=HEBO
```

Comment:
where is `huawei-noah` going? we do not need it anymore?

Reply:
we can use the pypi version now